WeakAuras/BossMods.lua

```diff
@@ -129,10 +128,19 @@ Private.ExecEnv.BossMods.DBM = {
         if bar.expirationTime < now then
           if bar.scheduledScanExpireAt == nil or bar.scheduledScanExpireAt <= GetTime() then
             self.bars[timerId] = nil
+          else
+            if self.nextExpire == nil then
+              self.nextExpire = bar.scheduledScanExpireAt
+            elseif bar.scheduledScanExpireAt < self.nextExpire then
```

Comment:
can't `bar.scheduledScanExpireAt` be nil here?

Reply:
No if it is nil (or <= GetTime()) we would be in line 130

Reply:
indeed sorry

your changes are good, i did only tests with auto-clone and didn't think enough about nextExpire